pkg/virt-launcher/virtwrap/device/sriov/hostdev_test.go

```diff
@@ -322,6 +359,17 @@ func (d deviceDetacherStub) DetachDeviceFlags(data string, flags libvirt.DomainD
 	return nil
 }
 
+type deviceAttacherFlagsStub struct {
```

Comment:
Please remove the `Flags` part as you did in the production code.

Reply:
I forgot to rename it back, I will change it